File: C++/1146.cpp
#include<iostream>

using namespace std;

int main(){
    int entrada, indice = 1;

    while(true){
        cin >> entrada;

        if(entrada == 0){
            break;
        }
        
        for(indice = 1; indice <= entrada; indice++){
            cout << indice;
            if(indice != entrada){
                cout << " ";
            }

        }
        cout << endl;
    }

    return 0;
}

File: JavaScript/1146.js
var input = require('fs').readFileSync('stdin', 'utf8');

var lines = input.split('\n');
let entrada, indice;
let i = 0;
let resultado = "";

while(true){
    entrada = parseInt(lines[i]);

    if(entrada === 0){
        break;
    }

    if(i != 0){
        resultado += `\n`;
    }

    for(indice = 1; indice <= entrada; indice++){
        if(indice === entrada){
            resultado += `${indice}`
        }else{
            resultado += `${indice} `
        }
    }
    i++;
}

console.log(`${resultado}`)